flags.html

```diff
@@ -4,80 +4,81 @@
         <link rel="stylesheet" href="flags.css">
     </head>
     <body>
-<div id="indonesia" class="flagContainer">
+        <div id="indonesia" class="flagContainer">
+            <div class="topSegment red"></div>
+            <div class="bottomSegment white"></div>
+        </div>
+        <div id="columbia" class="flagContainer">
 
-</div>
-<div id="columbia" class="flagContainer">
+        </div>
```

Comment:
watch out about having a prettifier in your code.  that can cause massive conflicts and unnecessary changes.  This is often from a plugin in your editor.

Reply:
OK Dan. I looked in my VS code and the only extensions I have are live server, open in browser, and Power Mode... Don't think I have a prettifier.